multi-node/aws/pkg/cluster/config.go

```diff
@@ -26,6 +43,9 @@ type Config struct {
 	WorkerCount              int    `yaml:"workerCount"`
 	WorkerInstanceType       string `yaml:"workerInstanceType"`
 	WorkerRootVolumeSize     int    `yaml:"workerRootVolumeSize"`
+	VpcCIDR                  string `yaml:"vpcCidr"`
+	SubnetCIDR               string `yaml:"subnetCidr"`
```

Comment:
We should probably call this `ServerCIDR`, `NodeCIDR`, or `InstanceCIDR` instead. We'll also need pod and service cidr options.